sample-lnd.conf

```diff
@@ -1413,6 +1413,9 @@
 ; Set to disable experimental endorsement signaling.
 ; protocol.no-experimental-endorsement=false
 
+; Set to disable support for RBF based coop close.
```

Comment:
* Set to enable support for RBF based coop close.

https://github.com/lightningnetwork/lnd/pull/9857